.travis.yml

```diff
@@ -4,7 +4,7 @@ dist: jammy
 sudo: required
 services: xvfb
 node_js:
-  - "20.16.0"
+  - "20.15.1"
```

Comment:
Travis CI struggles with bumping the Node version, often getting stuck during environment initialization. Clearing Travis CI caches didn't help.

Bumping the Node version isn't required since Meteor's built-in Node still supports the necessary Node version for testing. Also, we plan to eventually migrate all pipelines to GitHub Actions, which will ensure compatibility. I tested version 20.17.0 locally, and everything worked fine, with tests passing.

Reply:
Bummer. The sooner the migration happens the better.